client/src/reducers/privateChat.js

```diff
@@ -67,7 +67,7 @@ export default (state = Map(), action) => {
       return state.updateIn([reciepient, 'history'], l => {
```

Comment:
@bonham000 may be a little late for this for this PR, seeing as I just merged it, and it only matters if other people will eventually be looking at this code base, but I noticed a little typo here: `reciepient` should actually be `recipient`.